controllers/chat/sendSignedMessage.js

```diff
@@ -6,6 +6,7 @@ const Environment = require('../../config/environment');
 const {determineMessageType, processReplyMessage} = require('../../utils/chat');
 const UsersFunction = require('../../databases/functions/users');
 const {UserBlockedUser} = require('../../databases/models');
+const BetterSocialCore = require('../../services/bettersocial');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BetterSocialCore` module is imported but it's not clear if it's being used in this file. If it's not used, consider removing the import to improve code cleanliness and reduce unnecessary dependencies.
```diff
- const BetterSocialCore = require('../../services/bettersocial');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->